lib/signaling/v3/tracksubscriptionssignaling.js

```diff
@@ -29,6 +29,13 @@ class TrackSubscriptionsSignaling extends MediaSignaling {
             break;
         }
       });
+
+      // NOTE(mpatwardhan): we receive ready message every time
+      // MSP channel is established. That means at startup and and at every VMS-failover.
+      if (this._currentRevision !== null) {
+        log.warn('resetting current version after VMS failover', this._currentRevision);
```

Comment:
The client is handling the failover properly, maybe this should be 'info' rather than 'warn'?